fetch.bs

```diff
@@ -4725,9 +4728,7 @@ omitted from <a enum><code>RequestMode</code></a> as it cannot be used nor obser
 <p>A {{Request}} object has an associated
 <dfn id=concept-request-request for=Request export>request</dfn> (a <a for=/>request</a>).
 
-<p>A {{Request}} object also has an associated {{Headers}} object which
-is itself associated with <a for=Request>request</a>'s
-<a for=request>header list</a>.
+<p>A {{Request}} object also has associated <dfn for=Request>headers</dfn> (a {{Headers}} object).
```

Comment:
Also, should we say "null or a Headers object" and initially null since it's not always there? (Same for Response.)